models/aws_robomaker_retail_RetailShopFloor_01/model.sdf

```diff
@@ -12,6 +12,14 @@
             <scale>1 1 1</scale>
           </mesh>
         </geometry>
+        <surface>
+          <friction>
+            <ode>

```

Comment:
should we set friction values for bullet physics engine as well? do we have a equivalent baseline for bullet?